harbor_exporter.go

```diff
@@ -208,6 +214,36 @@ func NewExporter(opts harborOpts, logger log.Logger) (*Exporter, error) {
 		"Get public repositories which are accessed most.).",
 		[]string{"repo_name", "repo_id"}, nil,
 	)
+	replicationStatus = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, opts.instance, "replication_status"),
+		"Get status of the last execution of this replication policy: Succeed = 1, any other status = 0.",
+		[]string{"repl_pol_name"}, nil,
+	)
+	replicationsTotal = prometheus.NewDesc(
```

Comment:
can we merge these 4 metrics in to 1? like 
```
prometheus.BuildFQName(namespace, opts.instance, "replications_count_total"),
		"Get number of replication tasks in total in the last execution of this replication policy.",
		[]string{"type", "repl_pol_name"}, nil,)
```
I think it will be easy to build the grafana dashboard.

Reply:
Good idea. Maybe we make it 2 metrics? One for the status (0 or 1) and one for the 5 counters (total, succeed, failed, in_progress, stopped).

Reply:
yes

Reply:
changes done & tested & committed